__tests__/app.test.js

```diff
@@ -26,3 +27,23 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.result).toEqual(data.topicData)
```

Comment:
I'd recommend an array length check, forEach and toMatchObject for this test but this works.